pom.xml

```diff
@@ -26,6 +26,11 @@
             <version>5.9.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>4.8.1</version>
+        </dependency>
```

Comment:
You want to only have this dependency in `<scope>test</scope>`

Reply:
Removing for now testing is put on the backburner